src/initialize.js

```diff
@@ -128,6 +132,31 @@ export async function auth(requireUser, hydrateUser) {
   }
 }
 
+/*
+ * Set or overrides configuration through an API.
+ * This method allows runtime configuration.
+ * Set a basic configuration when an error happen and allow initError and display the ErrorPage.
+ */
+
+export async function runtimeConfig() {
+  try {
+    const apiConfig = { headers: { accept: 'application/json' } };
+    const apiService = await configureCache();
+
+    const url = parseUrlQueryParams(getConfig().MFE_CONFIG_API_URL, [
+      { name: 'mfe', value: (getConfig().PUBLIC_PATH.slice(1, -1) || window.location.host.split('.')[0]) },
+    ]);
+
+    const { data } = await apiService.get(url, apiConfig);
+    mergeConfig(data);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error('Error with config API', error.message);
+    setConfig({
```

Comment:
I _think_ we should let the MFE keep its build-time configuration rather than using `setConfig` here.  I think it'd be more correct to leave all the build-time config values in there, rather than removing everything but BASE_URL.  The way this is now, all the values will start returning `undefined` instead of whatever's in the build-time config, which will likely be reasonable fallbacks in many cases.